src/hand-tracking/src/utils.cpp

```diff
@@ -14,3 +14,48 @@ VectorXd hand_tracking::utils::axis_angle_to_euler
     AngleAxisd angle_axis(angle, axis);
     return angle_axis.toRotationMatrix().eulerAngles(angle_index_1, angle_index_2, angle_index_3);
 }
+
+
+VectorXd hand_tracking::utils::euler_to_axis_angle
```

Comment:
Can you please define an enum class like
```cpp
enum class AxisOfRotation
{
    UnitX,
    UnitY,
    UnitZ
};
```
and use it as input type instead of `std::size_t` to specify the euler angle configuration? In this way the code should be much more readable and less error prone. For example. What if an user passes `angle_index_1 = 3`?

Reply:
Done!